### PR TITLE
MIGRATION-6 - Adding LoginFailures model and course_name to CourseEnrollment.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR adds the LoginFailures model to the Student admin, also adds the course_name display field to the CourseEnrollmentAdmin.

## Notes:

In order to show the CourseEnrollment model in the admin page, you must enable the Waffle switch: student.courseenrollment_admin

## Previous work:

https://github.com/proversity-org/edx-platform/commit/6782f6a3b8b146b02db726b9087361232d6c3849
https://github.com/proversity-org/edx-platform/commit/743ffbd7203c00ec02bfb37f81712c0e25108b49
https://github.com/proversity-org/edx-platform/commit/d8236894880b9aa4b3d0a7715a78c8cac295183f
https://github.com/proversity-org/edx-platform/commit/45bf96608f42a56735e265791ad2b6d7556684ca

## Reviewers:

- [ ] @andrey-canon 
- [ ] @diegomillan

### CODE REVIEW

File: common/djangoapps/student/admin.py [hunk 1]
```diff
@@ -316,11 +320,20 @@ class Meta(object):
         model = CourseEnrollmentAllowed
 
 
+class LoginFailuresAdmin(admin.ModelAdmin):
+    """ Admin interface for the LoginFailures model """
+    list_display = ('user', 'failure_count', 'lockout_until')
```
Comment (andrey-canon) [inline common/djangoapps/student/admin.py]: trailing comma

**Review by andrey-canon — APPROVED** [review]

File: common/djangoapps/student/admin.py (hunk 1 above)
Comment (andrey-canon) [reply common/djangoapps/student/admin.py]: .

Comment (andrey-canon) [reply common/djangoapps/student/admin.py]: .

**Review by andrey-canon — CHANGES_REQUESTED** [review]

**Comment by Squirrel18** [comment]
PR addressed. @andrey-canon

**Review by andrey-canon — APPROVED** [review]

**Pull request merged**